libpod/container_api.go

```diff
@@ -714,3 +714,39 @@ func (c *Container) Restore(ctx context.Context, options ContainerCheckpointOpti
 	defer c.newContainerEvent(events.Restore)
 	return c.restore(ctx, options)
 }
+
+// Indicate whether or not the container should restart
+func (c *Container) ShouldRestart(ctx context.Context) bool {
```

Comment:
Did you duplicate all of this logic? It looks like it was copied from container_internal

Reply:
It is, I will use the same function in both places.